engine/src/main/java/org/terasology/network/internal/NetworkSystemImpl.java

```diff
@@ -847,7 +879,7 @@ private void connectClient(Client client) {
         }
         WorldGenerator worldGen = context.get(WorldGenerator.class);
         if (worldGen != null) {
-            serverInfoMessageBuilder.setReflectionHeight(worldGen.getWorld().getSeaLevel() + 0.5f);
+            serverInfoMessageBuilder.setReflectionHeight(worldGen.getWorld().getSeaLevel());
```

Comment:
Why are you changing reflection height as part of this PR?

Sometimes it is reasonable to make a quick-fix for an unrelated issue while working on a file, but whenever you do so, clearly indicate this in the main PR description, and since GitHub makes it easy to do so, add a GitHub code comment to the line to make it obvious to any reviewers or mergers of your code.

Reply:
It looks like it was a fairly recent change to the main repo and I haven't pulled it into my local workspace. I'll change it back.